sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTestBase.java

```diff
@@ -307,37 +308,37 @@ void deleteSettingWithETagRunner(BiConsumer<ConfigurationSetting, ConfigurationS
     @Test
     public abstract void listWithMultipleKeys();
 
-    void listWithMultipleKeysRunner(String key, String key2, BiFunction<ConfigurationSetting, ConfigurationSetting, List<ConfigurationSetting>> testRunner) {
+    void listWithMultipleKeysRunner(String key, String key2, BiFunction<ConfigurationSetting, ConfigurationSetting, IterableResponse<ConfigurationSetting>> testRunner) {
```

Comment:
Do we actually want to use `IterableResponse` as the response type of the function? Using `Stream` or `Iterable`, preferably `Iterable`, would be sufficient to verify the tests. It'd also allow the `List` -> `Stream` -> `Flux` -> `IterableResponse` in the async tests to be removed as that doesn't test anything.